docs/farming-&-staking/farming/additional-guides/gpu-plotter.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Experimental GPU Plotter
```

Comment:
Maybe it's just me, but the word "experimental" just doesn't sit right. Do you think that GPU plotter trial release, test release, pre-release, test build, preliminary release?